packages/client/src/logic/const.ts

```diff
@@ -9,6 +9,7 @@ import npcEnemyHuggy from '/src/assets/images/game-npc-huggy.png';
 import npcEnemyKissy from '/src/assets/images/game-npc-kissy.png';
 import collisionBood from '/src/assets/images/game-collision-blood.png';
 import collisionTeleport from '/src/assets/images/game-collision-teleport.png';
+import speed from '/src/assets/images/game-ico-speed.png';
```

Comment:
```suggestion
import speedometer from '/src/assets/images/game-icon-speedometer.png';
```
используй пожалуйста полные наименования чтобы в будущем не возникало сомнений о чем речь, так как фактически это не иконка скорости, а иконка спидометра)